src/applications/vaos/tests/new-appointment/redux/helpers/formSubmitTransformers.v2.unit.spec.js

```diff
@@ -113,7 +114,198 @@ describe('VAOS V2 data transformation', () => {
         },
         extension: { desiredDate: '2019-12-02T00:00:00+00:00' },
         locationId: '983',
-        comment: 'Follow-up/Routine: asdfasdf',
+        reasonCode: {
+          coding: [
+            {
+              code: 'Follow-up/Routine',
+            },
+          ],
+          text: 'asdfasdf',
```

Comment:
delete `text:` It is not required for radio options besides "My reason isn't listed"

Reply:
Done